PacketDotNet/DrdaDDMPacket.cs

```diff
@@ -110,37 +110,45 @@ public DrdaCodepointType CodePoint
         }
 
         private List<DrdaDDMParameter> paramters;
+
```

Comment:
You can fold these improvements into your initial commit if they are improving that commit to keep the history cleaner.

Reply:
And this applies for all improvement changes in here that could go into the initial commit.